Jenkinsfile

```diff
@@ -0,0 +1,15 @@
+node {
+	stage 'checkout'
+	git url: "https://github.com/azavea/raster-foundry.git", 
```

Comment:
From what I was able to find, this snippet allows reuse of the SCM configuration within Jenkins. Also more generic than `git`:

```groovy
checkout scm
```